data/abilities.ts

```diff
@@ -2356,6 +2357,9 @@ export const Abilities: {[abilityid: string]: AbilityData} = {
 					delete target.volatiles['slowstart'];
 					this.add('-end', target, 'Slow Start', '[silent]');
 				}
+				if (strongWeathers.includes(target.getAbility().id)) {
+					this.add('-end', target, target.getAbility().id, '[silent]');
```

Comment:
```suggestion
					this.singleEvent('End', this.dex.abilities.get(target.getAbility().id), target.abilityState, target, pokemon, 'neutralizinggas');
```
Should make it so you can drop changes in sim/field.ts